x-pack/solutions/observability/plugins/slo/server/services/summary_transform_generator/helpers/create_temp_summary.ts

```diff
@@ -34,11 +34,10 @@ export interface EsSummaryDocument {
   };
   // SLO definition
   slo: {
-    // >= 8.14: Add indicator.params on the temporary summary as well as real summary through summary pipeline
-    indicator: { type: IndicatorTypes } | Indicator;
+    indicator: { type: IndicatorTypes } | Indicator; // >= 8.14: Add indicator.params on the temporary summary as well as real summary through summary pipeline
     timeWindow: t.OutputOf<typeof timeWindowSchema>;
     groupBy: string | string[];
-    groupings: Record<string, unknown>;
+    groupings: Record<string, unknown>; // not flatten
```

Comment:
What does this mean? I feel like this comment may be a bit confusing in the future.

Reply:
I think it means that, for example, the service name is saved as 

`{ service: { name: "service-name" } }
`

instead of

`{ "service.name": "service-name" }`

in groupings.

It's a comment that Kevin added when starting working on this, I can write this example above if you think it'd make it clearer.